docs: update 5.16.0 release notes (#1466)
elastic/apm-agent-rum-js: CHANGELOG.asciidoc

[[release-notes-5.x]]
=== RUM JS Agent version 5.x

[[release-notes-5.16.0]]
==== 5.16.0 (2023-12-27)
[float]
===== Features

> * Report Interaction to Next Paint (INP) metric. This new metric, which replaces FID, will begin to be visible in the User Experience App in version 8.12.0 of the Elastic Stack. {pull}1462[#1462]

Reviewer: ```suggestion
* Add support for [Interaction to Next Paint (INP) metric](https://web.dev/articles/inp). This new metric, which replaces FID, will be available in the User Experience App from version 8.12.0 of the Elastic Stack. {pull}1462[#1462]
```

Author: done!

<img width="878" alt="Screenshot docs review" src="https://github.com/elastic/apm-agent-rum-js/assets/15065076/7934bf39-b579-4993-acf6-afea58703799">
